usb/esp_tinyusb/tusb_msc_storage.c

```diff
@@ -291,8 +306,21 @@ esp_err_t tinyusb_msc_storage_mount(const char *base_path)
         };
         cb(&event);
     }
+    uint32_t ticks_finish_cb_post = xTaskGetTickCount();
```

Comment:
I'm not sure I understand this logic - the code seem like single-thread sequence of calls, so there's a chance it gets blocked by the CB user completely... isn't it more appropriate to run a separate thread to watch the timeout?

Reply:
I am not able to find out the timing limitations by which callback should be finished both in case of mount and unmount of storage medium.
In case of S3, I observed it to be 3500ms in case of unmount and 10000ms in case of mount. But it may be different for other chips.

So, for now I am skipping this debug timing part from this PR.
I will revisit it later after releasing the current customer requirement (i.e. pre-unmount callback).

Reply:
Ok, thanks for the update